src/routes/propose-a-talk/+page.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import { onMount } from "svelte";
+
+	onMount(() => {
+		const hideElements = document.querySelectorAll('.hide');
+		let i = 0;
+		for (const el of hideElements) {
+			setTimeout(() => {
+				el.classList.add('show');
+			}, i * 400);
+			i++;
+		};
+	});
+</script>
+
+<section class="wrapper intro">
+	<h1 class="h2 hide">Propose a talk</h1>
+	<p class="h5 hide">We like all kinds of talks – whether it's about a framework you worked on, your first project with a new library, experimenting with new features, or a unique side project. Talks at Svelte Society can be up to 30 minutes, and newcomers are always welcome.</p>
+	<p class="hide">
+		<a class="button" target="_blank" href="https://forms.gle/8SR9Gn8mXNMxMzZy6">I want to give a talk</a>
+	</p>
+</section>
+
+<section class="wrapper prepare hide">
+		<h2>Prepare your talk</h2>
+		<p>Take the time to practice your talk. Improvising live on stage is nice, but have a well prepared backup.</p>
+		<h3>Here's what you need to know:</h3>
+		<ul>
+			<li>You can select one of the three talk length options. Make sure not to exceed this limit</li>
+			<li>Your talk must be in English</li>
+			<li>Your slides should be in the format 16:9</li>
```

Comment:
Minor thing, but we need 16:9? Or it's to comply with Youtube format?

Reply:
16:9 for the presentation screen ;)